core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -895,4 +899,4 @@ oppia.filter('summarizeDefaultOutcome', [
       return summary;
     };
   }
-]);
+]);s
```

Comment:
Goodness! Pardon me for not noticing that. I thought I swept it line by line. Thank you very much @anmolshkl